src/views/MyVideosView/MyVideosView.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useEffect, useState } from 'react'
+import { useVideos } from '@/api/hooks'
+import { VideoPreview } from '@/components'
+import { useDrafts } from '@/hooks'
+import { Grid, Pagination, Tabs } from '@/shared/components'
+import ActionBarMyVideos from './ActionBarMyVideos'
+import {
+  PaginationContainer,
+  StyledDismissibleMessage,
+  StyledText,
+  TabsContainer,
+  ViewContainer,
+} from './MyVideos.styles'
+
+const testChannelId = '100' // staging test channel id
+const TABS = ['All Videos', 'Published', 'Drafts', 'Unlisted']
+const INITIAL_VIDEOS_PER_ROW = 4
+// not yet doable
+// TODO: adjust action bar to the real fee
+// TODO: Unlisted videos
+// TODO: on edit video callbacks
+// TODO: on delete video callbacks
+// TODO: dynamic channels (not hardcoded)
+// TODO: No videos screen (this was deleted from figma?)
+// doable
+// TODO: how is the action bar supossed to work?
+export const MyVideosView = () => {
+  const [videosPerRow, setVideosPerRow] = useState(INITIAL_VIDEOS_PER_ROW)
+  // Drafts calls can run into race conditions
```

Comment:
Why is that? Possibly fixed by my other comment for `useDrafts`. We cannot have race conditions